api/controllers/apps_test.go

```diff
@@ -55,8 +60,11 @@ func TestAppShowWithAppNotFound(t *testing.T) {
 
 func TestAppCreate(t *testing.T) {
 	aws := test.StubAws(
-		test.CreateAppStackCycle("application"),
-		test.DescribeAppStackCycle("application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeStackNotFound("convox-test-application"),
+		test.CreateAppStackCycle("convox-test-application"),
+		test.DescribeStackNotFound("application"),
+		test.DescribeAppStackCycle("convox-test-application"),
```

Comment:
Ok, I'm starting to see it, based on the TestAppCreateWithAlreadyExistsBound test...